content/llm/speaking-bots.md

```
@@ -986,9 +983,24 @@ Each endpoint is documented with:
 - Response details
 - Example requests and responses
 
+All API requests require a MeetingBaas API key to be passed in the `x-meeting-baas-api-key` header.
+
 Use the navigation to explore the available endpoints.
 
 
```

markdownlint-cli2 (0.17.2)

Multiple consecutive blank lines
Expected: 1; Actual: 2

(MD012, no-multiple-blanks)